frontend/ui/widget/configdialog.lua

```diff
@@ -927,25 +934,31 @@ function ConfigDialog:onShowConfigPanel(index)
 end
 
 function ConfigDialog:onConfigChoice(option_name, option_value)
+    -- ToDO notification for activityIndicator check
     self.configurable[option_name] = option_value
     self.ui:handleEvent(Event:new("StartActivityIndicator"))
     return true
 end
 
-function ConfigDialog:onConfigEvent(option_event, option_arg, when_applied_callback)
+function ConfigDialog:onConfigEvent(option_event, option_arg, when_applied_callback, source)
     self.ui:handleEvent(Event:new(option_event, option_arg, when_applied_callback))
     return true
 end
 
-function ConfigDialog:onConfigEvents(option_events, arg_index)
+function ConfigDialog:onConfigEvents(option_events, arg_index, source)
     for i=1, #option_events do
         option_events[i].args = option_events[i].args or {}
         self.ui:handleEvent(Event:new(option_events[i].event, option_events[i].args[arg_index]))
     end
     return true
 end
 
-function ConfigDialog:onConfigChoose(values, name, event, args, events, position, hide_on_apply)
+function ConfigDialog:onConfigChoose(values, name, event, args, events, position, hide_on_apply, source)
+    if #values == 0 then -- this is a toggle which is not selectable (eg. increase, decrease)
+        Notification:setNotificationSource(Notification.SOURCE_BOTTOM_MENU_FINE)
```

Comment:
(and you can deal with this edge case in the only place(s) where this might happen.)